src/python/pants/option/option_types.py

```diff
@@ -40,6 +37,9 @@ class OptionsInfo:
 _MaybeDynamicT = Union[_DynamicDefaultT, _DefaultT]
 # The type of the `help` parameter for each option.
 _HelpT = _MaybeDynamicT[str]
+# NB: Ideally this would be `_RegisterIfFuncT`, however where this type is used is
+# generally provided untyped lambdas.
```

Comment:
I don't get this comment. "Ideally this would be .." and then it seems to be just that .. ?

Reply:
Ahh I think thats a bad find+replace